Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ImageBlendBrush/ImageBlendBrushXaml.bind

```diff
@@ -18,7 +18,7 @@
       <Border Grid.Column="0">
         <Border.Background>
           <media:ImageBlendBrush
-            Source="/SamplePages/DropShadowPanel/Unicorn.png"
+            Source="/SamplePages/Shadows/Unicorn.png"
```

Comment:
Can we apply the same change to the poor T-Rex, as well?  It's gone extinct on this page.

Reply:
Ah, good catch! Fixed in 6a46fb5feb54698458f32322e853bc9c12de01c2 😄

![image](https://user-images.githubusercontent.com/10199417/135094421-d59009ee-be34-4a76-bc14-a3f074e5f594.png)